pkg/events/marshalers.go

```diff
@@ -30,9 +30,7 @@ func (s *HTTPRequest) MarshalLog(addField func(key string, value interface{})) {
 	addField("http.url_details.path", s.Path)
 	addField("http.url_details.uri", s.URI)
 	addField("http.url_details.endpoint", s.Endpoint)
-	if s.Route != "" {
-		addField("http.route", s.Route)
-	}
```

Comment:
This change was missed in #602, this file should have been flagged as an autogenerated file.

Reply:
Fixed in #619